bot/src/bounded-contexts/analytics/domain/services/processors/TrendDataProcessor.ts

```
@@ -0,0 +1,287 @@
+    config: ChartConfiguration,
+    rawData: any[]
+  ): Promise<ChartData> {
+    
+    const granularity = config.displayOptions?.granularity || 'day';
+    
+    switch (granularity) {
+      case 'hour':
+        return this.processHourlyTrend(rawData, config);
+      case 'week':
+        return this.processWeeklyTrend(rawData, config);
+      case 'day':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)